localisation/map.csv

```diff
@@ -1523,7 +1523,7 @@ RHU_580;Gathod;x
 RHU_584;Morad;x
 RHU_590;Urgathavuld;x
 RHU_600;Urgov;x
-RHU_601;Rhunear Isles;x
+RHU_601;Rh�naer Isles;x
```

Comment:
Would you update name in the comment in map/regions.txt? The generator pulls the name from there